### Aufgabe 12:
Löse die folgenden Aufgaben in Python und speichere das Ergebnis unter *VornameNachname*-aufgabe12.ipynb auf Laufwerk H:.

1. Gegeben seien drei rechtwinklige, gleichschenklige Dreiecke. Die Katheten seien jeweils 1cm, 3cm und 7,99cm lang. Berechnen Sie die Hypothenusen.

In [10]:
katheten = [1,3,7.99]
for kathete in katheten:
    print("Die Hypothenuse für Kathete",kathete,"ist",2*kathete*kathete)

Die Hypothenuse für Kathete 1 ist 2
Die Hypothenuse für Kathete 3 ist 18
Die Hypothenuse für Kathete 7.99 ist 127.68020000000001


### Lösung:

`Die Hypothenuse für Kathete 1 ist 2`<br>
`Die Hypothenuse für Kathete 3 ist 18`<br>
`Die Hypothenuse für Kathete 7.99 ist 127.68020000000001`<br>